bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/discovery/HDPowerViewShadeDiscoveryService.java

```diff
@@ -78,7 +78,7 @@ private Runnable createScanner() {
             try {
                 shades = targets.getShades();
             } catch (IOException e) {
-                logger.error("{}", e.getMessage(), e);
+                logger.error("Unexpected error: {}", e.getMessage());
```

Comment:
```suggestion
                logger.warn("Unexpected error: {}", e.getMessage());
```

Can the binding recover from that?

Reply:
Ok. Done.